app/scripts/lib/AddressBookPetnamesBridge.ts

```diff
@@ -35,14 +52,10 @@ export class AddressBookPetnamesBridge extends AbstractPetnamesBridge {
     const entries: PetnameEntry[] = [];
     const { state } = this.#addressBookController;
     for (const chainId of Object.keys(state.addressBook)) {
-      // TODO: Replace `any` with type
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      const chainEntries = state.addressBook[chainId as any];
+      const chainEntries = state.addressBook[chainId as Hex];
 
       for (const address of Object.keys(chainEntries)) {
-        // TODO: Replace `any` with type
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const entry = state.addressBook[chainId as any][address];
+        const entry = state.addressBook[chainId as Hex][address];
```

Comment:
```suggestion
        const entry = state.addressBook[chainId][address];
```

Reply:
Question: is it a good idea to remove type here since the TODO specifies `// TODO: Replace `any` with type`?
and the chainId is not typed as Hex yet

Reply:
By calling `!isStrictHexString(chainId)` we are making sure the chainId is of `Hex` type. If you check the type of chainId after the calling the `if(!isStrictHexString(chainId))` you can see that the type of chainId is now Hex or `0x{string}`